components/component_storage/tests/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+include(HPX_Option)
```

Comment:
Is this needed here? Shouldn't be the target-specific modules be included instead?

Reply:
Not strictly needed. Do you mean by target-specific modules CMake modules that are in `components/X/cmake`? Including this here would make sense if we wanted to build components standalone, but I'm not sure that makes sense (at least right now) so I'll remove it for now. The modules do the same thing here to be able build them standalone, but that still needs testing. Also, the modules are subprojects while the components are not. And the modules set `CMAKE_MODULE_PATH` so that it could actually work standalone. All in all, components are not ready for this yet, so I'll remove it.